### PR TITLE
added css battle and frontendmentor on challenges/games section

### PR DESCRIPTION
**Pull request by moeen-mahmud**
Hi!

I added CSS Battle and Frontend Mentor in the Challenges/Games Section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -573,10 +573,12 @@ Native,UI/UX and cool Designing stuff ![Free](https://img.shields.io/badge/-Free
...
 - [CSS Zen Garden](http://www.csszengarden.com/) - A demonstration of what can be accomplished through CSS-based design, and a chance to do your own. ![Free](https://img.shields.io/badge/-Free-brightgreen)
 - [Flexbox Defense](http://www.flexboxdefense.com) - It is a game that covers the flex properties align-items, justify-content, flex-direction, align-self and has 12 different levels. ![Free](https://img.shields.io/badge/-Free-brightgreen)
 - [Flexbox Froggy](https://flexboxfroggy.com) - This game is also about Flexbox and it covers even more flex properties: align-items, justify-content, align-content, flex-direction, align-self, flex-wrap, flex-flow and has 24 different levels. ![Free](https://img.shields.io/badge/-Free-brightgreen)
+- [Frontend Mentor](https://www.frontendmentor.io/solutions) - Frontend Mentor challenges help you improve your coding skills by building realistic projects. It is excellent for those who want to revisit their frontend knowledge. You will get a huge community if you get stucked in the challenges. ![Free](https://img.shields.io/badge/-Free-brightgreen)
```
Comment (RitikPatni) [inline README.md:581]: Please also add a paid badge on this, as front-end mentor also has paid problems.

**Review by RitikPatni — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (moeen-mahmud) [reply README.md]: I've added the paid badge. Please review.

**Comment by RitikPatni** [comment]
@moeen-mahmud Can you please resolve merge conflicts also please remove front-end mentor as well as I had another PR which also had front-end mentor, and I've already merged it.

**Review by RitikPatni — APPROVED** [review]

**Comment by moeen-mahmud** [comment]
I've removed the frontend mentor from the list.

**Pull request merged**